tests/participant/test_alias_models.py

```diff
@@ -0,0 +1,97 @@
+from dataservice.extensions import db
+from dataservice.api.participant.models import Participant
+from dataservice.api.study.models import Study
+from tests.utils import FlaskTestCase
+
+
+class ModelTest(FlaskTestCase):
+    """
+    Test database model
+    """
+
+    def _create_save_to_db(self):
+        """
+        Create participant with external id
+        """
+        s = Study(external_id='phs001')
+        p = Participant(external_id='Fred', family_id="Test_family_id_0",
+                        is_proband=False, consent_type="GRU-IRB")
+        s.participants.append(p)
+        db.session.add(s)
+        db.session.commit()
+
+        alias_data = {
+            'Fred1': {
+                'external_id': 'Fred1',
+                'is_proband': False
+            },
+            'Fred2': {
+                'external_id': 'Fred2',
+                'is_proband': False
+            }
+        }
+        for _, kwargs in alias_data.items():
```

Comment:
Is this an accurate portrayal of a use case? Fred1 and Fred2 are aliases of Fred, but Fred1 is not an alias of Fred2.

Reply:
It would be if I implemented a method to retrieve all aliases for a participant through a dfs. Right now aliases are basically modeled as an undirected graph, so I could probably just change the way aliases are retrieved and return all aliases of a participant through dfs instead of just the immediate aliases. I don't think we'd want to store alias links for all participants that are aliased to each other bc that could quickly grow large.